ons_alpha/bulletins/models.py

```diff
@@ -61,22 +57,25 @@ class BulletinPage(BasePage):
         ]
     )
 
-    def get_context(self, request, *args, **kwargs):
-        context = super().get_context(request, *args, **kwargs)
-
-        context["toc"] = self.toc
-
-        return context
+    @property
+    def is_latest(self):
+        latest_id = (
+            BulletinPage.objects.sibling_of(self).live().order_by("-release_date").values_list("id", flat=True).first()
+        )
+        return self.pk == latest_id
 
-    @cached_property
-    def toc(self):
-        items = [{"url": "#summary", "text": "Summary"}]
 
-        for block in self.body:  # pylint: disable=not-an-iterable
-            if hasattr(block.block, "to_table_of_contents_items"):
-                items += block.block.to_table_of_contents_items(block.value)
+class BulletinSeriesPage(RoutablePageMixin, Page):
```

Comment:
Now that the new page model is in the correct place, you need to make a new migration as per my previous comment.

The check for missing migrations is failing - 
https://github.com/ONSdigital/dis-wagtail-alpha/actions/runs/9896847664/job/27339953666?pr=20:

```
Run poetry run ./manage.py makemigrations --check --noinput
Skipping virtualenv creation, as specified in config file.
Migrations for 'bulletins':
  ons_alpha/bulletins/migrations/0003_bulletinseriespage_alter_bulletinpage_body_and_more.py
    - Create model BulletinSeriesPage
    - Alter field body on bulletinpage
    - Alter field updates on bulletinpage
```

You need to run `python manage.py makemigrations` (but after you undo the MathJax change), please.